extensions/wikia/FacebookClient/FacebookClient.setup.php

```diff
@@ -34,7 +34,6 @@
 $wgAutoloadClasses['FacebookMapModel'] =  $dir . 'FacebookMapModel.class.php';
 $wgAutoloadClasses['FacebookClientFactory'] =  $dir . 'FacebookClientFactory.php';
 $wgAutoloadClasses['FacebookClientController'] =  $dir . 'FacebookClientController.class.php';
-$wgAutoloadClasses['SpecialFacebookConnectController'] =  $dir . 'SpecialFacebookConnectController.class.php';
```

Comment:
Line 66 of this file should also be removed https://github.com/Wikia/app/pull/6026/files#diff-ac942f59224da105aff611a08a8dfe5dR66